qpm.json

```diff
@@ -4,7 +4,7 @@
   "info": {
     "name": "SongBrowser",
     "id": "songbrowser",
-    "version": "0.1.0",
+    "version": "1.0.2",
```

Comment:
We don't actually update our versions directly, we let the GH action on release do that, this is not needed